sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -268,6 +268,44 @@ def create_oct_key(self, name, **kwargs):
         hsm = kwargs.pop("hardware_protected", False)
         return self.create_key(name, key_type="oct-HSM" if hsm else "oct", **kwargs)
 
+    @distributed_trace
+    def create_okp_key(self, name: str, **kwargs: "Any") -> KeyVaultKey:
```

Comment:
I don't know enough about Python but I couldn't help to notice that you are specifying the data type for the name and kwargs arguments, as opposed to what's done in other create_key operations, why is that?